sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md

```diff
@@ -1,15 +1,25 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 2.0.0 (2022-11-07)
```

Comment:
Please don't bump major version for track 2.

Reply:
@fengzhou-msft - Changed version number by bumping minor version number.